### ISSUE TITLE
[CLDR] Edit a currency display by language (2/3)

### ISSUE THREAD

**Issue by MatShir** [PrestaShop/PrestaShop#14750]
**USER STORY**
As a merchant, I want to modify the currency format if needed in order to adapt my shop to international norms.

**WHY**
The CLDR data doesn't always answer all the merchant's needs, and we've seen lots of tickets asking for currency modifications (not available in 1.7 so far).

- Here the first part #10052 and the related PR #10455

**SPECIFICATION**
This new page lists of the currency display according to all the languages of the shop with the possibility to edit them.

Add/Edit currency display by language:
- The symbol
- The symbol position around the amount

**DETAILED SPECIFICATION**

[Format Setting](https://github.com/PrestaShop/prestashop-specs/blob/master/core/back-office/Localization-Currencies-List-Add-Edit.md#currency-format-setting)

the first part of the CLDR Front End development -> https://github.com/nihco2/PrestaShop/tree/feat/currencies-page-rework

**DESIGN**
Link to the Invision models:
https://invis.io/SAPYJZCCR6K

The popup will display "Apply" to update the format on the row of languages table instead of "Save".

**ISSUES RELATED**
- #9714 
- #10014
- #10017 
- #10055
- #10054
- #10053
- #9885
- #10021

**OLD SPECIFICATION**
- #10058
- #10057

Blocked by [#15752 ](https://github.com/PrestaShop/PrestaShop/issues/15752)
RAF : 0.5 day

**Comment by go-sudo** [comment]
Hello 
i tried to change the currency title from MAD to DHS on PS 1.7.6.0

In order to change or edit the currency symbol, you need to use the FTP or cPanel File Manager of your hosting server.
Go to translations/cldr/mail-xx-XX-currencies file (where xx-XX is your language, en-EN, de-DE or ru-RU). Open this file with any code editor.
Find there the following code:
```
"USD": {
"displayName": "US Dollar", "displayName-count-one": "US dollar", "displayName-count-other": "US dollars", 
"symbol": "$", "symbol-alt-narrow": "$"
	}
```

But still same issue

Regards

**Comment by go-sudo** [comment]
Hello @MatShir 

Any news about this issue

regards

**Comment by matks** [comment]
@go-sudo The work is in progress: https://github.com/PrestaShop/PrestaShop/pull/14972

**Comment by colinegin** [comment]
Hello @jolelievre , is this issue fixed by #14972 ?

**Comment by jolelievre** [comment]
Hi everyone!
No, the PR #14972 is just the first part linked to this issue #15752
But the work for this issue is well advanced in a branch, I just didn't create a PR because other ones need to be managed first. Also (as the numerotation doesn't indicate) this issue will have to be merged first https://github.com/PrestaShop/PrestaShop/issues/14709
